webpack.config.ts

```diff
@@ -26,13 +26,13 @@ var commonConfig = {
     // add 'var CompressionPlugin = require("compression-webpack-plugin");' on the top
     // and comment out below codes
     //
-    // new CompressionPlugin({
-    //   asset: "[path].gz[query]",
-    //   algorithm: "gzip",
-    //   test: /\.js$|\.css$|\.html$/,
-    //   threshold: 10240,
-    //   minRatio: 0.8
-    // })
+    // , new CompressionPlugin({
```

Comment:
@gdi2290 Thank you for the quick reply. Have made the change and pushed it. 
I hope this is what you meant by a trailing comma , please inform if the change is not as expected :)